src/Data/Detection/DetectionRepository.cs

```diff
@@ -0,0 +1,36 @@
+namespace Data.Detection
+{
+    using System.Collections.Generic;
+    using Business.Detection.Common.Models;
+    using Business.Detection.Common.Repositories;
+
+    public class DetectionRepository : IDetectionRepository
+    {
+        private readonly Context _context;
+
+        public DetectionRepository(Context context)
+        {
+            _context = context;
+        }
```

Comment:
Should we convert this constructor into a one liner?

Reply:
Done